src/DOM/rendering.ts

```diff
@@ -76,6 +76,13 @@ function removeRoot(root: Root): void {
 	}
 }
 
+if (isBrowser && document.body === null) {
+	if (process.env.NODE_ENV !== 'production') {
```

Comment:
I'm going to change the throwError API so that it does the production check inside the function, can you change your PR and remove the `if (process.env.NODE_ENV === 'production')` check to change which error message is sent?